docs/source/onedal/spmd/index.rst

```diff
@@ -0,0 +1,97 @@
+.. ******************************************************************************
+.. * Copyright 2020-2021 Intel Corporation
```

Comment:
```suggestion
.. * Copyright 2021 Intel Corporation
```

Reply:
It was just an attempt to try exact copy of license from existing file